circe/src/test/scala/org/http4s/circe/CirceSpec.scala

```diff
@@ -370,5 +364,6 @@ class CirceSpec extends JawnDecodeSupportSpec[Json] with Http4sLegacyMatchersIO
     }
   }
 
-  checkAll("EntityCodec[IO, Json]", EntityCodecTests[IO, Json].entityCodec)
+// TODO
+//  checkAll("EntityCodec[IO, Json]", EntityCodecTests[IO, Json].entityCodec)
```

Comment:
@cquiroz: @benderpremier and I got stuck here today, because there's no `Eq[IO[A]]` anymore.  I think this is the same issue you had on boopickle where we another test integration library?  Or can we get this done now just by porting to MUnit?

Reply:
I think we need a `discipline-cats-effect` that provides a `checkAllF` but we are not there yet.
As a workaround I made a `BoopickleSuite` with the `munit` tests and kept `BoopickleSpec` with only the law check

Reply:
I thought that's what you did, and I went looking for it yesterday.  I think I was just on the wrong branch.

Reply:
Oh, is that not on cats-effect-3 yet?  I was thinking this is what needed to move to munit, but it's the thing you held _out_ of munit.  But I'm not sure how it runs on Specs2, because we're missing the old `Eq[IO[A]]` that we used to have.